src/Plugin/resource/DataProvider/DataProviderEntity.php

```diff
@@ -484,15 +469,22 @@ protected function getEntityInfo($type = NULL) {
    */
   protected function getQueryForList() {
     $query = $this->getEntityFieldQuery();
-    if ($path = $this->request->getPath()) {
-      $ids = explode(',', $path);
-      if (!empty($ids)) {
-        $query->entityCondition('entity_id', $ids, 'IN');
-      }
+
+    // If we are trying to filter or sort on a computed field, just ignore it
+    // and log an exception.
```

Comment:
We are providing sort by ID by default, but, for entities, ID is a computed field (`getIdentifier()`). I feel that it is a very common case to have computed fields.

I'm not too sold on this either, but I don't see a clear exit…